framework/src/base/Assembly.C

```diff
@@ -2116,26 +2130,38 @@ Assembly::reinitNeighborFaceRef(const Elem * neighbor,
 }
 
 void
-Assembly::reinitLowerDElemRef(const Elem * elem,
-                              const std::vector<Point> * const pts,
-                              const std::vector<Real> * const weights)
+Assembly::reinitLowerDElem(const Elem * elem,
+                           const std::vector<Point> * const pts,
+                           const std::vector<Real> * const weights)
 {
-  mooseAssert(pts->size(),
-              "Currently reinitialization of lower d elements is only supported with custom "
-              "quadrature points; there is no fall-back quadrature rule. Consequently make sure "
-              "you never try to use JxW coming from a fe_lower object unless you are also passing "
-              "a weights argument");
-
   _current_lower_d_elem = elem;
 
   unsigned int elem_dim = elem->dim();
+  mooseAssert(elem_dim < _mesh_dimension,
+              "The lower dimensional element should truly be a lower dimensional element");
+
+  if (pts)
+  {
+    // Lower rule matches the face rule for the higher dimensional element
+    ArbitraryQuadrature * lower_rule = _holder_qrule_arbitrary_face[elem_dim + 1];
+
+    // This also sets the quadrature weights to unity
+    lower_rule->setPoints(*pts);
+
+    if (weights)
+      lower_rule->setWeights(*weights);
+
```

Comment:
`weights` is a passed-in parameter. How do you make sure the sum of them is unity? You have a comment:

`// This also sets the quadrature weights to unity`

Reply:
We cannot do an a priori check here. The sum of the weights should be equal to the area of the reference element and the same quadrature rule may be being applied to several different element types. I think we just have to trust the user that if they're going to pass in weights (which is currently not done anywhere in the framework) that they know what they're doing